src/ess/isissans/sans2d.py

```diff
@@ -22,15 +22,19 @@
 """Sample holder mask"""
 
 
-def detector_edge_mask(sample: TofData[SampleRun]) -> DetectorEdgeMask:
+# It may make more sense to depend on CalibratedDetector here, but the current
```

Comment:
Yes, that would make more sense, but we need the masks to be there before we start computing a center of mass, which is annoying.

Reply:
The masks are anyway applied after setting the beam center (see your other comment https://github.com/scipp/esssans/pull/155#discussion_r1696831864). So this could easily be changed, but maybe that is not how the scientists work? Do you remember where these numbers come from?

Reply:
Note than in principle we *could* set masks before loading data (when they come from a file).... but maybe that is not desirable if other types of masks come into play?

Reply:
Numbers for masks came from trial and error with Wojciech. We just wanted to mask the edges and found values that worked.